frontends/web/src/components/balance/balance.tsx

```diff
@@ -73,7 +70,7 @@ export const Balance = ({
               {' '}{balance.incoming.unit} /
               <span className={style.incomingConversion}>
                 {' '}
-                <FiatConversion amount={balance.incoming} noBtcZeroes/>
+                <AmountWithUnit amount={balance.incoming} removeBtcTrailingZeroes convertToFiat/>
```

Comment:
Small regression when sending, I believe due to the `unstyled` prop that was there before.

before this commit:
<img width="320" alt="Screenshot 2025-03-12 at 12 14 23" src="https://github.com/user-attachments/assets/132c2099-3547-49a9-b029-5fed1bb699f7" />

with this commit the BTC unit is very large:
<img width="411" alt="Screenshot 2025-03-12 at 12 14 00" src="https://github.com/user-attachments/assets/181eb2b5-549f-4936-9613-82a56c01f418" />

Reply:
Nice catch! Thanks :pray: